### ISSUE TITLE
Top-level destructured bindings violate spacing rule

### ISSUE THREAD

**Issue by bgourlie** [avh4/elm-format#266]
`elm-format` will incorrectly format the following snippet:

```
{ id, class, classList } =
    CssCommon.helpers


type alias Model =
    { mnemonic : String
    , operand : String
    , offset : Int
    }
```

After running `elm-format`, it becomes:

```
{ id, class, classList } =
    CssCommon.helpers
type alias Model =
    { mnemonic : String
    , operand : String
    , offset : Int
    }
```

**Comment by mattrrichard** [comment]
The workaround I'm using for this issue is to put a comment between the two declarations.  It's not great, but at least it looks a little less bad:
```
{ id, class, classList } =
    CssCommon.helpers

--


type alias Model =
    { mnemonic : String
    , operand : String
    , offset : Int
    }
```

Is "stable" for me.

**Comment by avh4** [comment]
Fixed in 9679b7a...fa7344f.  Will be included in 0.5.1-alpha
